scripts.js

```diff
@@ -0,0 +1,28 @@
+// console.log('hey')
+
```

Comment:
My default preference is to not have commented out code in PRs unless there is a really good reason. A lot of this seems superfluous.